code/modules/surgery/surgery_step.dm

```diff
@@ -82,7 +82,7 @@
 	if(implement_type)	//this means it isn't a require hand or any item step.
 		implement_speed_mod = implements[implement_type] / 100.0
 
-	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod)
+	speed_mod /= (get_location_modifier(target) * (1 + get_internals_modifier(target)) * (1 + surgery.speed_modifier) * implement_speed_mod)

```

Comment:
Is the goal here to reward only those using proper anesthetic tanks from medbay or being sedated during surgery?

If it's the latter and the goal is to reward being vulnerable during surgery, it would be much better to check for sources of being knocked out than to do a snowflake check for a singular group of tank typepaths.